VideoWeb/VideoWeb/ClientApp/src/app/shared/error/error.component.spec.ts

```diff
@@ -97,6 +97,16 @@ describe('ErrorComponent', () => {
         expect(component.isExtensionOrFirewallIssue).toBeFalsy();
     });
 
+    it('should show default error message if internet connection is down', () => {
+        spyPropertyGetter(connectionStatusServiceSpy, 'status').and.returnValue(false);
+
+        component.ngOnInit();
+        expect(component.errorMessageTitle).toBe("There's a problem with your connection");
```

Comment:
single quotes - might fail linting otherwise

Reply:
I missed it. Resolved it now

Reply:
@phillipjchambers I had to revisit the code. Fixing linting introduced issues with prettify